### ISSUE TITLE
Send Notifications to Your Android Application

### ISSUE THREAD

**Issue by plazhbanov** [SAPDocuments/Tutorials#3672]
Tutorial URL: https://developers.sap.com/tutorials/cp-sdk-android-wizard-app-push.html

Please specify the step you are referring to - Service configuration. On the Roles tab, select the Notification User role and click the Assign button, assigning your User ID.

I'm trying to set up the role for a notification user in the Cloud Foundry environment, but I don't see the same page in the cockpit. Could you please help me?

**Comment by plazhbanov** [comment]
Hi Dear, 
Do you have any ideas?

**Comment by thecodester** [comment]
@jarjarbentley Is this something you can help with?

**Comment by jarjarbentley** [comment]
We have reached out to Dirk Grosskopf on the push services team for help via sap email.

**Comment by thecodester** [comment]
@jarjarbentley Any update?

**Comment by danielva** [comment]
I believe the documentation for push using Mobile Services for Cloud Foundry has been updated.  See [Defining Push Notifications](https://help.sap.com/viewer/468990a67780424a9e66eb096d4345bb/Cloud/en-US/82864e6309d84aed9566436b766e63ef.html).

I don't believe the step of assigning a user to the role Notification User applies.

I was able to create an app using the SAP Cloud Platform SDK for Android wizard that used Mobile Services on Cloud Foundry.  In the Mobile Services cockpit, under the "Mobile Push Notification" feature, under the configuration tab, add the server key and sender id (from the Google Firebase console).  Then start the mobile app to complete a registration.  Then within the Mobile Services cockpit, under the "Mobile Push Notification" feature, under the Push Registrations tab, click on Send Notification to the user you wish to receive the notification.

Hope that helps.
Dan van Leeuwen
